projects/addon-commerce/pipes/decimal/decimal.options.ts

```diff
@@ -0,0 +1,17 @@
+import type {Provider} from '@angular/core';
+import type {TuiCurrencyVariants} from '@taiga-ui/addon-commerce/types';
+import {tuiCreateToken, tuiProvideOptions} from '@taiga-ui/cdk/utils/miscellaneous';
+
+export interface TuiDecimalOptions {
+    readonly currency: TuiCurrencyVariants;
```

Comment:
I don't think we need options for that. I believe most of the world places currency before the number and we do not need this pipe in our internal designs.